crates/sui-core/src/consensus_types/consensus_output_api.rs

```diff
@@ -89,3 +90,59 @@ impl ConsensusOutputAPI for narwhal_types::ConsensusOutput {
             }).collect()
     }
 }
+
+impl ConsensusOutputAPI for mysticeti_core::consensus::linearizer::CommittedSubDag {
+    fn reputation_score_sorted_desc(&self) -> Option<Vec<(AuthorityIndex, u64)>> {
+        // TODO: Implement this in Mysticeti.
+        None
+    }
+
+    fn leader_round(&self) -> u64 {
+        self.anchor.round
+    }
+
+    fn leader_author_index(&self) -> AuthorityIndex {
+        self.anchor.authority as AuthorityIndex
+    }
+
+    fn commit_timestamp_ms(&self) -> u64 {
+        // TODO: Enforce ordered timestamp in Mysticeti
+        0
```

Comment:
We can still return timestamp from the leader block for now (I am not sure if anything can break in sui if timestamp is always 0)

Reply:
> timestamp from the leader block for now

But that's not enforced to be monotonic, right? That would be worse than returning 0?

Reply:
It's not enforced, but I think it is very unlikely